src/features/hotspots/root/HotspotsScreen.tsx

```diff
@@ -49,7 +49,7 @@ const HotspotsScreen = () => {
             flexDirection: 'column',
           }}
         >
-          <Box style={{ flex: 1.5 }} justifyContent="center">
+          <Box style={{ flex: 1.8 }} justifyContent="center">
```

Comment:
I don't see how this would address Wayne's problem mentioned in the issue where the bottom of the screen is cut off: https://nebraltd.slack.com/archives/C0XGF69SN/p1648681771596559?thread_ts=1647279479.807949&cid=C0XGF69SN

Reply:
Have you refreshed recently? I updated the description 5 days ago. Also check additional text below the image.

Reply:
@marvinmarnold the PR was targeted towards fixing the issue reported by Kelvin with regards to part of the circular button not showing. 

I feel both issues ("Sometimes the top of the screen is cut off" and "Sometimes the bottom of the screen is cut. This is especially problematic when it makes it impossible to link your wallet") are not necessarily related and have to be separated. If you agree, i will create a new issue for "Sometimes the bottom of the screen is cut. This is especially problematic when it makes it impossible to link your wallet"

Reply:
Sure that works

Reply:
Please as discussed i have created a separate issue for ""Sometimes the bottom of the screen is cut. This is especially problematic when it makes it impossible to link your wallet". Kindly review this.

Reply:
https://github.com/NebraLtd/maker-starter-app/issues/92